scripts/partition_manager/partition_manager.rst

```diff
@@ -774,21 +774,24 @@ You can set ``PM_STATIC_YML_FILE`` to contain exactly the static configuration y
 
 If you do not set ``PM_STATIC_YML_FILE``, the build system will use the following order to look for files in your application source directory to use as a static configuration layout:
 
-* If build type is used (see :ref:`modifying_build_types`), the following order applies:
+* If a custom configuration is used (see :ref:`modifying_build_types`), the following order applies:
 
-  1. If the file :file:`pm_static_<board>_<revision>_<buildtype>.yml` exists, it will be used.
-  #. Otherwise, if the file :file:`pm_static_<board>_<buildtype>.yml` exists, it will be used.
-  #. Otherwise, if the file :file:`pm_static_<buildtype>.yml` exists, it will be used.
+  1. If the file :file:`pm_static_<board>_<revision>_<file_suffix>.yml` exists, it will be used.
```

Comment:
as above, this is not implemented yet

Reply:
Moved the note to before this paragraph and restored the previous wording. This way, the two approaches should be clearly differentiated.

Reply:
@nordicjm can you elaborate on this a bit more?

I'm very confused, as if this is true that such a feature is not supported i see this PR as a big regression - and documentation may suggest not working commands. However if it still works (which my colleague seems confirm) then the comment is not clear for me. Appreciate your help here.

Reply:
It does not work, it is not implemented, it will be implemented when sysbuild ncs is picked up again (after the current upmerge). You can use any file suffix you want, it will use `pm_static.yml` because PM has not been updated to use this new variable.

Reply:
@nordicjm can you elaborate on this a bit more?

I'm very confused, as if this is true that such a feature is not supported i see this PR as a big regression - and documentation may suggest not working commands. However if it still works (which my colleague seems confirm) then the comment is not clear for me. Appreciate your help here.

Reply:
Okay, then are we in hurry with this PR? Shouldn't we first fully migrate to sysbuild?

If we want to proceed with this PR, at least I would ensure that in all places we explicitly in the doc specify the PM_STATIC_FILE and many samples based on the current auto selection mechanism.

Reply:
I'm OK with waiting until after the sysbuild transition

Reply:
I have reverted changes to Matter and some other areas, while keeping the general additions about file suffixes (and in the samples that got the approval).